webapp/src/components/modals/create_issue/create_issue.jsx

```diff
@@ -240,23 +239,47 @@ export default class CreateIssueModal extends PureComponent {
             </React.Fragment>
         );
 
-        if (jiraIssueMetadata && jiraIssueMetadata.error) {
+        let issueError = null;
+        let component;
+
+        // if no getMetaDataError, show fields and allow user to input
+        // fields. An error at this point is from a server-side create
+        // issue submission and should be displayed (in addition to fields)
+        // to the user after clicking the create button.
+        if (error) {
+            issueError = (
+                <p className='alert alert-danger'>
+                    <i
+                        className='fa fa-warning'
+                        title='Warning Icon'
+                    />
+                    <span> {error}</span>
+                </p>
+            );
+        }
+
+        // if getmetadata fails, only display the error and a close button.
+        // user is not going to be able to create a ticket or even a partial
+        // ticket. likely a permissions error.
+        if (getMetaDataError) {
+            const metaDataError = (
+                <p className='alert alert-danger'>
+                    <i
+                        className='fa fa-warning'
+                        title='Warning Icon'
+                    />
+                    <span> {getMetaDataError}</span>
+                </p>
+            );
+            component = metaDataError;
+            footer = footerClose;
+        } else if (jiraIssueMetadata && jiraIssueMetadata.error) {
```

Comment:
Probably nothing you touched here, but I'm confused about this -- is it possible to have a null getMetaDataError, but a jiraIssueMetadata.error?

Reply:
@cpoile Actually, there is a case where jiraIssueMetadata.error is returned from the server. This occurs when getting Jira metadata returns without any projects. This was introduced from a ticket a while back.    

We could probably refactor this use one method for returning all errors.  

For reference, `getMetadataError` catches the following returned errors `issue.go`

```go
		return http.StatusInternalServerError,
			errors.WithMessage(err, message)
```

A `jiraIssueMetadata.error` is returned from the following in `issue.go`
```go
	var bb []byte
	if len(cimd.Projects) != 0 {
		bb = []byte(`{"error": "You do not have permission to create issues in any projects. Please contact your Jira admin."}`)
	} else {
		bb, err = json.Marshal(cimd)
		if err != nil {
			return http.StatusInternalServerError,
				errors.WithMessage(err, "failed to marshal response")
		}
	}
	_, err = w.Write(bb)

```

Reply:
I don't have a preference on which method use, but perhaps we should only use one?  Also not sure if I should address in this ticket?